### PR TITLE
Update base python/debian image and optimize image size

### PR DESCRIPTION
**Pull request by evgkrsk**
I think that 90+ critical CVE in base image is too much :-)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -8,9 +8,12 @@ RUN apt-get update && apt install -y locales libcurl4-openssl-dev libssl-dev \
 
 WORKDIR /app/
 
-RUN pip install poetry==1.2.1
+RUN pip install poetry==1.2.1 \
+    && rm -rf /root/.cache
```
Comment (danihodovic) [inline Dockerfile:12]: Why is /root/.cache being removed?

File: Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM python:3.9.1
+FROM python:3.9.14-bullseye
```
Comment (danihodovic) [inline Dockerfile:1]: Can we upgrade this to 3.12?

File: Dockerfile (hunk 1 above)
Comment (evgkrsk) [reply Dockerfile:14]: To decrease final images size by ~100Mb

File: Dockerfile (hunk 2 above)
Comment (evgkrsk) [reply Dockerfile]: In my tests, "poetry install" fails with python 3.11 and 3.12 images (bullseye) :-(

File: Dockerfile [hunk 3]
```diff
@@ -1,16 +1,21 @@
-FROM python:3.9.1
+FROM python:3.10.9-bullseye
 
 EXPOSE 9808
 
-RUN apt-get update && apt install -y locales libcurl4-openssl-dev libssl-dev \
+RUN apt-get update && \
+    apt-get -y dist-upgrade && \
```
Comment (evgkrsk) [inline Dockerfile:6]: "dist-upgrade" here to apply latest debian security updates to close critical CVEs.

**Comment by danihodovic** [comment]
Спасибо Евгений.

Released as danihodovic/celery-exporter:0.6.1

**Pull request merged**